Makefile.am

```diff
@@ -43,7 +43,7 @@ export:
 	    echo `date`: Using existing tarball: $(TARFILE);			\
 	else									\
 	    rm -f $(PACKAGE).tar.*;                                             \
-	    (git archive --prefix=$(distdir)/ $(shell echo $(TAG)|cut -f1 -d-) || tar -c --transform="s,^,$(distdir)/," --exclude="*.tar.*" --exclude="$(distdir)" --exclude="*.o" --exclude="*.8" --exclude="config.*" --exclude="Makefile" --exclude="Makefile.in" --exclude="stamp-*" --exclude="*.service" --exclude="sbd" --exclude="*.m4" --exclude="*.cache" --exclude="configure" --exclude="*.list" --exclude="depcomp" --exclude="install-sh" --exclude="missing" --exclude="compile" --exclude="sbd.sh" --exclude="~" --exclude="*.swp" --exclude="*.patch" --exclude="*.diff" --exclude="*.orig" --exclude="*.rej" --exclude="*.rpm" --exclude=".deps" --exclude="test-driver" *) | gzip > $(TARFILE);	\
+	    (git archive --prefix=$(distdir)/ $(shell echo $(TAG)|cut -f1 -d-) || tar -c --transform="s,^,$(distdir)/," --exclude="*.tar.*" --exclude="$(distdir)" --exclude="*.o" --exclude="*.8" --exclude="config.*" --exclude="libtool" --exclusive="ltmain.sh*" --exclude="Makefile" --exclude="Makefile.in" --exclude="stamp-*" --exclude="*.service" --exclude="sbd" --exclude="*.m4" --exclude="*.cache" --exclude="configure" --exclude="*.list" --exclude="depcomp" --exclude="install-sh" --exclude="missing" --exclude="compile" --exclude="sbd.sh" --exclude="~" --exclude="*.swp" --exclude="*.patch" --exclude="*.diff" --exclude="*.orig" --exclude="*.rej" --exclude="*.rpm" --exclude=".deps" --exclude="test-driver" *) | gzip > $(TARFILE);	\
```

Comment:
You could automate this with something like

    IGNORE = $(shell grep -v '#' .gitignore)
    EXCLUDES=$(foreach ignored,$(IGNORE),--exclude "$(ignored)")

though I haven't tested that syntax

Reply:
And you'd have to make sure to distribute .gitignore

Reply:
Putting .gitignore in a tarball might be confusing.
Ideal would be a common source for both but I don't know if you can feed .gitignore from a file.

Reply:
Don't see a way how to include another file into .gitignore.
Git has a bunch of possibilities to exclude files but as .gitignore is the standard I wouldn't like to use any of the other that aren't that common.
As I'd like to use 'git archive' I don't see a way to package .gitignore into that (by e.g. changing the name to something that doesn't sound git-specific preferrably).
So I guess I'll go the duplication-route once again thoping to have a better idea when it changes next time.